src/partners.js

```diff
@@ -0,0 +1,44 @@
+"use strict";
```

Comment:
@ryyppy why is this not in `.gitignore`?

Reply:
not sure... we have to manually ignore the `.js` files, since we cannot use a `bs.js` prefix in gatsby